src/caseStudies/CollegeAdmissionsUT.js

```diff
@@ -3,7 +3,7 @@ import "../main.css";
 import teachLogo from "../assets/teachla-logo.svg";
```

Comment:
oops i guess it was supposed to be `AppMain.css` instead. can you please change it back? i think I tried changing it to `main.css` before but then it caused some issues I couldn't figure out how to solve so I changed it back to `AppMain.css`